### DISCUSSION TITLE
Minecart hopper not receiving or distributing resources? 1.16.4

### DISCUSSION THREAD

**Discussion by Sylith23** [GriefPrevention/GriefPrevention#1191] [Q&A] [answered]
I'm currently trying to make an autosmelter within my claim, and my chests/furnaces/hoppers are privated. My resources can not go from the chest into the minecart hopper. Is there a solution?

**Comment by bigpresh** [comment]
I don't believe anything GriefPrevention does would interfere with the operation of hoppers.

I'd imagine it's some other plugin you're running - can you see if the problem persists with just GriefPrevention running?

If so, please raise a ticket as a bug report, filling out the issue template: https://github.com/TechFortress/GriefPrevention/issues/new?assignees=&labels=Unverified+Bug&template=bug-report.md&title=

**Reply by Sylith23** [reply]
I figured out the command as /cremove , it had changed from /c unprivate   so that's why I was having a problem.

**Reply by bigpresh** [reply]
That's not a GriefPrevention command - it sounds like you are using [LWC](https://dev.bukkit.org/projects/lwc) and that's what was blocking it.

I'm not entirely sure why you'd use both GriefPrevention and LWC together, but it wasn't GP causing the issue.

**Comment by Jikoo** [comment, marked as answer]
Private chests (and your problem) are from LWC, not GriefPrevention. GP does not currently handle hopper minecarts' interactions at all.
